src/util.cpp

```diff
@@ -1415,8 +1415,7 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
     if (!comments.empty())
         ss << "(" << boost::algorithm::join(comments, "; ") << ")";
     ss << "/";
-    ss << "Primecoin:" << FormatVersion(PRIMECOIN_VERSION);
-    ss << "(" << CLIENT_BUILD << ")/";
```

Comment:
this line is useful in that it shows the client is not official build (the custom build version string will show in getpeerinfo

Reply:
e.g. CLIENT_BUILD is the part in paranthesis: 
Satoshi:0.8.6/Primecoin:0.1.2(v0.1.2xpm-92-gde59af1-beta)/